app/models/BannerTests.scala

```diff
@@ -7,9 +7,10 @@ import io.circe.{Decoder, Encoder}
 
 import scala.collection.immutable.IndexedSeq
 
-
 sealed trait BannerTemplate extends EnumEntry
-object BannerTemplate extends Enum[BannerTemplate] with CirceEnum[BannerTemplate] {
+object BannerTemplate
+    extends Enum[BannerTemplate]
+    with CirceEnum[BannerTemplate] {
```

Comment:
I think we'll probably want to drop the formatting changes in this file

Reply:
Think this is a consequence of format on save, do we have lint rules for these scala files or would I need to disable my IDE settings to prevent changes?

Reply:
We don't have any linting setup for scala. Scala devs seem to not like it so much 🤷‍♂️

Reply:
Ah okay, no worries